### DISCUSSION TITLE
Incompatibility between old and new layout!

### DISCUSSION THREAD

**Discussion by niklas2810** [firefly-iii/firefly-iii#5957] [Firefly III] [answered]
Hi James, I've just installed Firefly 5.7.0 and love the new layout so far! And as you wrote, definitely faster progress than on v2 😄 However, after trying out v3 and switching back to v1, I receive the following error:

![image](https://user-images.githubusercontent.com/30748216/161427834-e7a259af-52bc-41de-99e1-7cd2e6d339c4.png)

I guess this is because the new and old layout don't have a matching data format. Unfortunately, this bricked my entire installation, so not only in my current browser, but on other instances as well. Do you regard to this as a basic incompatibility or a bug? If the latter is the case, I'll be happy to open an issue with the debug site output.

**Comment by niklas2810** [comment]
The issue is in `updateStartDate` of `Navigation.php`. `rangeChoices` of DateRange.vue doesn't seem to be compatible with the old layout 😅

**Reply by JC5** [reply]
I pushed some new values, it'll be part of the new release :+1:

**Comment by JC5** [comment, marked as answer]
Yeah that's a risk, sorry about that.

**Reply by niklas2810** [reply]
Alright, no worries! Guess that's the risk you're taking when testing out unfinished features 😄 I've edited the values by hand now 😅

**Comment by niklas2810** [comment]
⚠️ In case someone else has the same issue ⚠️

1. Log in to your database (MySQL/Postgres) and login to the database (`USE your_FireflyIII_db_name;`)
2. Show the preferences table  (`SELECT * FROM preferences;`)
3. There should be a value `viewRange` -> `last30` in there. If not, that's a different issue!
4. Update this value to a valid one! (`UPDATE preferences SET data='1M' WHERE name='viewRange';`)
5. If you have updated your `FIREFLY_III_LAYOUT` in the past, you might have to revert this back to `v1` too, for now 😅Don't know why, but at least in my installation this caused some trouble as well.
